src/clp_ffi_js/ir/decoding_methods.cpp

```diff
@@ -33,4 +42,54 @@ auto rewind_reader_and_validate_encoding_type(clp::ReaderInterface& reader) -> v
         };
     }
 }
+
+auto get_version(clp::ReaderInterface& reader) -> std::string {
```

Comment:
Style violation: Functions/method definitions should be ordered in the same order as their declarations.